subworkflows/local/input_check.nf

```diff
@@ -2,40 +2,39 @@
 // Check input samplesheet and get read channels
 //
 
-include { SAMPLESHEET_CHECK } from '../../modules/local/samplesheet_check'
```

Comment:
If you are not using this module anymore, you can also remove the files from `modules/local/samplesheet_check`

Reply:
un-used code and modules were removed.